### PR TITLE
Support any linked post type

### PR DESCRIPTION
**Pull request by cliffordp**
https://central.tri.be/issues/102120

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -35,6 +37,11 @@ We're always interested in your feedback and our [premium forums](https://theeve
 
 == Changelog ==
 
+= 2.1.0 2018-03-21 =
+
+* Feature - Regisers a new `[tec_list_linked_posts]` shortcode that can have any linked post type passed to its "post_type" argument, which means this shortcode now supports custom linked post types (see code comments for limitations), such as from https://theeventscalendar.com/knowledgebase/linked-post-types/
```
Comment (barryhughes) [inline readme.txt]: Typo: regisers

File: tribe-ext-list-venues-organizers-shortcodes.php [hunk 2]
```diff
@@ -97,116 +103,162 @@ protected function format() {
...
-					$args['organizer'] = $id;
-				} else {
-					$args['venue'] = $id;
+				if ( Tribe__Events__Main::ORGANIZER_POST_TYPE == $this->atts['post_type'] ) {
+					$args['organizer'] = $post_id;
+				} elseif ( Tribe__Events__Main::VENUE_POST_TYPE == $this->atts['post_type'] ) {
+					$args['venue'] = $post_id;
 				}
 
+				$args = apply_filters( 'tribe_ext_list_venues_organizers_shortcodes_tribe_get_events_per_post_args', $args, $post_id, $this->atts );
```
Comment (barryhughes) [inline tribe-ext-list-venues-organizers-shortcodes.php]: Unless this was an existing hook that was relocated, can we shorten it a little by removing the second use of 'tribe_'?

File: tribe-ext-list-venues-organizers-shortcodes.php [hunk 3]
```diff
@@ -97,116 +103,162 @@ protected function format() {
...
 				$events_count = count( $events );
 
 				// generate HTML
-				if ( 'yes' != $hide_empty || ( 'yes' == $hide_empty && $events_count > 0 ) ) :
-
-					$item       = '<li id="' . esc_attr( $this->atts['post_type'] . '-' . $id ) . '" class="tec list ' . $this->atts['post_type'] . '">';
+				if (
+					'yes' != $hide_empty
+					|| (
+						'yes' == $hide_empty
```
Comment (barryhughes) [inline tribe-ext-list-venues-organizers-shortcodes.php:128]: Formatting here or the next line is off

**Review by barryhughes — CHANGES_REQUESTED** [review]
Couple of minor tweaks and this should be good to go!

File: tribe-ext-list-venues-organizers-shortcodes.php (hunk 3 above)
Comment (cliffordp) [reply tribe-ext-list-venues-organizers-shortcodes.php:128]: @barryhughes Why is this formatting not correct?

Comment (barryhughes) [reply tribe-ext-list-venues-organizers-shortcodes.php:128]: We're using an initial indent of tabs on line 128 (correct) but a mix of tabs and spaces on line 129 (incorrect). I could have been clearer first time, though: the _following_ line needs to be fixed.

Comment (cliffordp) [reply tribe-ext-list-venues-organizers-shortcodes.php:128]: gotcha (I think). published regardless ;)

**Pull request merged**